src/tools/_confgen/generate_rst.c

```diff
@@ -26,6 +26,8 @@ static const char *xlatmd(const char *str, const char **end)
     { "<li>", " * " }, { "</li>", "\n" },
     { "<ul>", "" }, { "</ul>", "\n" },
     { "<sup>", "^" }, { "</sup>", "" },
+    { "<code>", "`" }, { "</code>", "`" },
```

Comment:
Rst code formatting should be double backticks, like this: `\`\`code\`\``

Reply:
Fixed in 9fb41da